components/src/hardware-sim/RobotCoordinateSpace/RobotCoordinateSpace.tsx

```diff
@@ -1,30 +1,31 @@
 import { animated } from '@react-spring/web'
-import styled from 'styled-components'
 
-import { Svg } from '../../primitives'
+import styles from './robotCoordinateSpace.module.css'
```

Comment:
i researched about the naming after tuesday's meeting and using the lowercase only is the best.

```suggestion
import styles from './robotcoordinatespace.module.css'
```